extensions/amp-viz-vega/0.1/amp-viz-vega.js

```diff
@@ -186,7 +202,9 @@ export class AmpVizVega extends AMP.BaseElement {
 
     return parsePromise.then(chartFactory => {
       return vsyncFor(this.win).mutatePromise(() => {
-        dom.removeChildren(this.container_);
+        const container = /** @type {!Element} */(
+            dev().assert(this.container_));
```

Comment:
use `dev().assertElement` instead, then you can get rid of the type cast annotation